Source/Extensions/Blazorise.SignaturePad/SignaturePadJSOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Blazorise.Modules.JSOptions;
+
+namespace Blazorise.SignaturePad;
+
+public class SignaturePadInitializeJSOptions
```

Comment:
Remove the `*Initialize*` from the name. All initialized options should be named `ComponentJSOptions`.